### ISSUE TITLE
ui: reorganize tiles to look nicer

### ISSUE THREAD

**Issue by meling** [quickfeed/quickfeed#1135]
We could maybe tweak the tiles a bit to make it look nicer/fancier.
- [ ] A simple fix would be to move the semester/year to a separate line in the box.
- [ ] Maybe (not sure) make the tile boxes narrower?

**Comment by JosteinLindhom** [comment]
Current:
![image](https://github.com/user-attachments/assets/eb13ecdc-36e9-4796-a1bf-600ba8fe25bc)

First attempt:
![image](https://github.com/user-attachments/assets/f5671bc3-21c0-4ce8-b74e-4b5009507e46)

**Comment by meling** [comment]
I think this looks nicer!! Let's go with this for now.

**Comment by meling** [comment]
yeah, I was wondering about that, but assume they were the same size.
